devlab/tests/generate_load.py

```diff
@@ -146,11 +151,63 @@ def run_preparation_scenario(self):
         self.modify_quotas()
         self.create_flavors()
         self.upload_image()
+        self.create_networks()
         self.create_vms()
         self.create_vm_snapshots()
-        self.create_networks()
+
+    def clean_objects(self):
+        for tenant in config.tenants:
+            try:
+                self.keystoneclient.tenants.delete(
+                    self.get_tenant_id(tenant['name']))
+            except Exception:
```

Comment:
Maybe it makes sense to print original exception? Cause It can be hard to find the original reason of error.